docs/sources/datasources/_index.md

```diff
@@ -91,6 +107,24 @@ These built-in core data sources are also included in the Grafana documentation:
 - [Testdata]({{< relref "./testdata" >}})
 - [Zipkin]({{< relref "./zipkin" >}})
 
+## Add additional data source plugins
+
+You can add additional data sources as plugins (that are not available in core Grafana), which you can install or create yourself.
+
+### Find data source plugins in the plugin catalog
+
+To view available data source plugins, go to the [plugin catalog](/grafana/plugins/?type=datasource) and select the "Data sources" filter.
+For details about the plugin catalog, refer to [Plugin management]({{< relref "../administration/plugin-management/" >}}).
+
+You can further filter the plugin catalog's results for data sources provided by the Grafana community, Grafana Labs, and partners.
+If you use [Grafana Enterprise]({{< relref "../introduction/grafana-enterprise/" >}}), you can also filter by Enterprise-supported plugins.
```

Comment:
This link will also need to be updated into a `docs/reference` style link like the others in the page because the linked "Administration" content doesn't exist in Grafana Cloud documentation.

For an example, refer to https://github.com/grafana/grafana/pull/83712/files#diff-7b86f2d305e1ba40bef37131931d4213ee72232f66190c6a31cc1de8b4d913ffR129-R130.

Reply:
This now fixed, please check

Reply:
The other is indeed fixed thanks for doing that :) Could you do the same with this one.

Reply:
I am not sure which link you meant? is it about the Grafana Enterprise link?

Reply:
Yeah that's the one. Sorry for the confusion, I copy-pasted my comment from the first instance and didn't update the link text.

Reply:
This is now fixed !! :slightly_smiling_face: